server/src/main/java/com/thoughtworks/go/server/service/PipelineHistoryService.java

```diff
@@ -553,17 +547,20 @@ private void populateMaterialRevisionsOnBuildCause(PipelineInstanceModel model)
         model.setMaterialRevisionsOnBuildCause(materialRepository.findMaterialRevisionsForPipeline(model.getId()));
     }
 
-    public void updateComment(String pipelineName, int pipelineCounter, String comment, Username username, HttpLocalizedOperationResult result) {
+    public void updateComment(String pipelineName, int pipelineCounter, String comment, Username username) {
```

Comment:
This method is used in the old comment API. Do we plan to remove that API or else we might have to update it.

Reply:
Yes, I am planning to remove that API. If not then I will update the rails controller

Reply:
Can we do that in this PR itself, else the build might fail.

Reply:
Done